jena-fuseki2/jena-fuseki-ui/src/services/fuseki.service.js

```diff
@@ -157,6 +158,15 @@ class FusekiService {
     return axios.get(this.getFusekiUrl('/$/tasks'))
   }
 
+  async listGraphs (datasetName, endpoint) {
+    const result = await axios.get(this.getFusekiUrl(`/${datasetName}/${endpoint}`), {
+      params: {
+        query: DATASET_GRAPHS
+      }
+    })
+    return ["default", ...result.data.results.bindings.map(binding => binding.g.value)]
```

Comment:
Cleaner than the other methods doing the `forEach`! Will apply that to the rest of the code in a future PR. Thanks!